jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc

```diff
@@ -0,0 +1,387 @@
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], the following is a summary of the TCK results for Jakarta EE Core Profile 11.
+
+== Open Liberty 24.0.0.11-beta Jakarta EE Core Profile 11 using Java 17 Certification Summary
+
+* Product Name, Version and download URL (if applicable):
++
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/beta/24.0.0.11-beta/openliberty-24.0.0.11-beta.zip[Open Liberty 24.0.0.11-beta]
+
+* Specification Name, Version and download URL:
++
+https://jakarta.ee/specifications/coreprofile/11[Jakarta EE Core Profile 11]
+
+* TCK Version, digital SHA-256 fingerprint and download URL:
++
+https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee11/staged/eftl/jakarta-core-profile-tck-11.0.0.zip[Jakarta EE Core Profile TCK 11.0 (Staged Release)],
+SHA-256: `cd5d039adcb17a626845ee43c03e4ff11fbdec6738afb251bc3b1778c389e1c2`
+
+* Public URL of TCK Results Summary:
++
+link:24.0.0.11-beta-Java17-TCKResults.html[TCK results summary]
+
+* Any Additional Specification Certification Requirements:
++
+Jakarta Annotations
+https://download.eclipse.org/jakartaee/annotations/3.0/jakarta-annotations-tck-3.0.0.zip[jakarta-annotations-tck-3.0.0.zip],
+SHA-256: `9421c6ca66274d32dfb408848f75a42d57f120599fe0d8403c5c5c1141d5ac4d`
++
+Jakarta Contexts and Dependency Injection
+https://download.eclipse.org/jakartaee/cdi/4.1/cdi-tck-4.1.0-dist.zip[cdi-tck-4.1.0-dist.zip],
+SHA-256: `446029ee1ce694d2a9ae8893d16be7afd7e1c0ed8705064b7095af174cf97ea0`
++
+Jakarta Dependency Injection
+https://download.eclipse.org/jakartaee/dependency-injection/2.0/jakarta.inject-tck-2.0.2-bin.zip[jakarta.inject-tck-2.0.2-bin.zip],
+SHA-256: `23bce4317ca061c3de648566cdf65c74b57e1264d6891f366567955d6b834972`
++
+Jakarta JSON Binding
+https://download.eclipse.org/jakartaee/jsonb/3.0/jakarta-jsonb-tck-3.0.0.zip[jakarta-jsonb-tck-3.0.0.zip],
+SHA-256: `954fd9a3a67059ddeabe5f51462a6a3b542c94fc798094dd8c312a6a28ef2d0b`
++
+Jakarta JSON Processing
+https://download.eclipse.org/jakartaee/jsonp/2.1/jakarta-jsonp-tck-2.1.1.zip[jakarta-jsonp-tck-2.1.1.zip],
+SHA-256: `949f203de84deffa8c7892b555918e42f1dd220ccb7b6800741ea58af62737c1`
++
+Jakarta RESTful Web Services
+https://download.eclipse.org/jakartaee/restful-ws/4.0/jakarta-restful-ws-tck-4.0.1.zip[jakarta-restful-ws-tck-4.0.1.zip],
+SHA-256: `b6290c1b5b3d2fdd9cc700a999243492a7e27b94a9b6af1974ff4dc5bfbf98f2`
+
+
+* Java runtime used to run the implementation:
++
+----
+java version "17.0.12" 2024-07-16
+IBM Semeru Runtime Certified Edition 17.0.12.1 (build 17.0.12+7)
+Eclipse OpenJ9 VM 17.0.12.1 (build openj9-0.46.1, JRE 17 Linux amd64-64-Bit Compressed References 20240716_775 (JIT enabled, AOT enabled)
+OpenJ9   - 4760d5d320
+OMR      - 840a9adba
+JCL      - 9882134b8c0 based on jdk-17.0.12+7)
+----
+
+* Summary of the information for the certification environment, operating system, cloud, ...:
++
+Apache Derby 10.14.2.0
```

Comment:
Which bit of core profile needed Derby?

Reply:
That's a good catch. Nothing in core does, that's vestigial copypasta from Platform/Web Profile docs.